thali/thalireplicationmanager.js

```diff
@@ -6,6 +6,7 @@ var inherits = require('util').inherits;
 var net = require('net');
 var multiplex = require('multiplex');
 var validations = require('./validations');
+var addressPrefix = 'addressbook-'
```

Comment:
Fixed as part of the redesign where the thalireplicationmanager does not add anything o the DB.